### PR TITLE
nrf_modem doc update for nRF9161

### PR DESCRIPTION
**Pull request by eivindj-nordic**
Depends on #1079

### CODE REVIEW

File: nrf_modem/doc/delta_dfu.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ Delta update packages
 
 A delta update is a binary file, with a :file:`.bin` extension.
 The update file is named in such a way that both the source and destination firmware versions can be identified, for example, :file:`mfw_nrf9160_update_from_1.2.2_to_1.2.3`.
-Modem delta update files are part of the official modem releases and can be found in the `nRF9160 modem firmware zip file`_.
+Modem delta update files are part of the official modem releases and can be found in the `nRF91 Series modem firmware zip file`_.
```
Comment (lemrey) [inline nrf_modem/doc/delta_dfu.rst:25]: I don't think we can simply rename the links, there are going to be separate links for modem firmware zip files.

File: nrf_modem/doc/sockets.rst [hunk 2]
```diff
@@ -353,10 +353,10 @@ Packet data networks
 
 The Modem library supports selecting which Packet Data Network (PDN) to use on a network socket and for DNS queries.
 The configuration of Packet Data Protocol (PDP) contexts, and the activation of PDN connections are not handled by the Modem library.
-To configure PDP contexts and activate PDN connections, the application must use the nRF9160 modem packet domain AT commands.
+To configure PDP contexts and activate PDN connections, the application must use the nRF91 Series modem packet domain AT commands.
 
 When performing network operations on any PDN, the application ensures that the PDN connection is available.
-For more information about how to configure PDP contexts, activate PDN connections, and determine their state, see the `nRF9160 modem Packet Domain AT commands`_ documentation.
+For more information about how to configure PDP contexts, activate PDN connections, and determine their state, see the `nRF91 Series modem Packet Domain AT commands`_ documentation.
```
Comment (lemrey) [inline nrf_modem/doc/sockets.rst:359]: There will also be separate AT command manuals.

Comment (divipillai) [reply nrf_modem/doc/sockets.rst:359]: Agree. I will be fixing this in the commit that I am working for the PR.

**Review by divipillai — APPROVED** [review]

File: nrf_modem/README.rst [hunk 3]
```diff
@@ -3,7 +3,7 @@
 Modem library
 #############
 
-The Modem library is the primary interface for operating the nRF9160 modem, establishing the LTE-M and NB-IoT connections, and receiving the position data (GPS).
+The Modem library is the primary interface for operating the cellular modem, establishing the LTE-M and NB-IoT connections, and receiving the position data (GPS).
```
Comment (lemrey) [inline nrf_modem/README.rst:6]: Could we have said the `nRF91 series modem` here?

**Review by lemrey — APPROVED** [review]
Thank you, just a minor comment.

File: nrf_modem/README.rst (hunk 3 above)
Comment (eivindj-nordic) [reply nrf_modem/README.rst:6]: @divipillai

Comment (divipillai) [reply nrf_modem/README.rst:6]: Agree, we could use the "nRF91 Series modem" term here. Could see similar terms used in the infocenter docs for nRF9160.

Comment (eivindj-nordic) [reply nrf_modem/README.rst]: Updated

**Pull request merged**